bin/i18n/sync-out.rb

```diff
@@ -23,11 +23,12 @@
 
 module I18n
   module SyncOut
-    def self.perform(upload_manifests: false)
+    def self.perform
       puts "Sync out starting"
+      I18n::Resources::Apps.sync_out
       rename_from_crowdin_name_to_locale
```

Comment:
When we download translations from Crowdin, the files are placed in folders with the full lanaguage and coutry name, for example: Spanish-Spain, Spanish-Mexico, etc.
THe first step in the sync out is to rename the long language name with the short language code, for example: es-ES,  es-MX

Reply:
https://github.com/code-dot-org/code-dot-org/pull/53436/files#diff-124524b3691f6d10195d792cb0f9f65f7c190335ccd4b7900a5741c9574df638R29-R31

Reply:
This step will happen for every single module, which could be abstracted as a separate method, as it currently exists. However, I see further in the sync out, that we iterate over locales more times like:
- When [restoring content.](https://github.com/code-dot-org/code-dot-org/pull/53436/files#:~:text=%5D-,Parallel,-.each()
- When [distributing content](https://github.com/code-dot-org/code-dot-org/pull/53436/files#:~:text=2%7D%20processes%22-,Parallel,-.each().

For each resource, will be iterating over each language once. Not sure if that will be more work that iterating over each language a few times for all resources.